manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
         privileged: false
         readOnlyRootFilesystem: true
 {{- end }}
-      serviceAccountName: istio-egressgateway-service-account
+      serviceAccountName: {{ $gateway.name | default "istio-egressgateway" }}-service-account
```

Comment:
At some point we should stop adding -service-account to objects that are already strongly typed.

Reply:
But the real issue is - I'm not sure that we want to create separate SA for each  egress or ingress gateway instance. 

In reality in a namespace the protection between accounts is relatively small - in particular if the accounts are granted Secret RBAC permission in that NS. 

If there is any strong reason to have separate SA  (and Role and RoleBinding) for each ingress - we can discuss. I don't see any real security benefit.

Reply:
I think this is just a merge conflict and already changed in master